### ISSUE TITLE
Starting with v6.8.3, TWiLight Menu++ now reads .sav files from the `saves` folder, which is in the exact same folder as the .nds ROMs.

### ISSUE THREAD

**Issue by Brenden535** [RocketRobz/TWiLightMenu#519]
Starting with v6.8.3, TWiLight Menu++ now reads .sav files from the `saves` folder, which is in the exact same folder as the .nds ROMs.
Move your existing .sav files in the .nds folder, to `saves` folder, if you haven't done so yet.

_Originally posted by @RocketRobz in https://github.com/RocketRobz/TWiLightMenu/issues/417#issuecomment-464417536_ 

Im quoting this because RocketRobz, I need help when I put my saves in the saves folder it doesnt recognize it and creates its own save in the saves folder, how do i fix this?

**Comment by RocketRobz** [comment]
You have to move your .sav files to the `saves` folder.
The app itself does not move them.

**Comment by RocketRobz** [comment]
Are you sure you're on the latest TWiLight Menu++ version?

**Comment by Brenden535** [comment]
Ill check

**Comment by Brenden535** [comment]
7.4.1 is latest right?

**Comment by RocketRobz** [comment]
Yes.

You moved each and every one of your .sav files into `saves`, right?
If you did, where is the `saves` folder? If it's not in the exact folder with the .nds files, then that's why TWLMenu++ is showing that a save file is created.

**Comment by Brenden535** [comment]
The saves folder is in the folder with the .nds files.

**Comment by Brenden535** [comment]
![c5cb048be4c8ee3cb5d34133a737165f](https://user-images.githubusercontent.com/49077212/55256871-3e5f9180-522c-11e9-9d68-32857551d748.png)

**Comment by RocketRobz** [comment]
You moved each and every one of your .sav files into `saves`, right?
Are you sure didn't miss any one of them?

**Comment by RocketRobz** [comment]
Which games showed that it created a save file?

**Comment by Brenden535** [comment]
Pokemon Volt White 2

**Comment by Brenden535** [comment]
![f982d2db5fb250f0a69486e8b8d8f8c4](https://user-images.githubusercontent.com/49077212/55256931-6f3fc680-522c-11e9-8a93-19958a82ef95.png)

**Comment by Brenden535** [comment]
The file that was created was Pokemon Volt White 2 - Complete.sav

**Comment by Brenden535** [comment]
The one I put in originally is Pokemon Volt White Play Through.sav

**Comment by RocketRobz** [comment]
So just rename `Pokemon Volt White Play Through.sav` to `Pokemon Volt White 2 - Complete.sav`.
Also, you might want to fix the date on your DSi or 3DS.
It's showing the .sav is created/modified in 2007.

**Comment by Brenden535** [comment]
Ok, thank you.

**Comment by Brenden535** [comment]
It worked, thank you so much!

**Comment by RocketRobz** [comment]
No problem!

**Comment by LordeJoey** [comment]
Hi, sorry I know this is closed but is the most similar problem I have found. I was playing Pokemon HeartGold on Drastic emulator on my cellphone and I edited some boxes on Pkhex, that I wanted to transfer to TwilightMenu++ on the DSi, in wich I haven't started a play through in Pokemon HeartGold, so I had to start again but my idea was that save I edited on Pkhex would make me start from where I last saved in Drastic, but that didn't happened. The Pkhex file is a .sav file that I copy into the saves folder in .nds ROMS but it doesn't seem to work. Can anyone help me out with this? I would really apreciated.
